docs/playbooks/how-to-get-started/index.md

```diff
@@ -0,0 +1,65 @@
+# How to get started as a developer/development team
+
+## What are the CPS Development Principles
+
+We follow the Central Digital and Data Office [Technology Code of Practice](https://www.gov.uk/guidance/the-technology-code-of-practice#be-open-and-use-open-source)
+
+## What environment do I use for development, and how do I get one?
+
+All development work at CPS must be performed on CPS-provided laptops. To install the required tools on your laptop, you will need local admin access.
+
+To request admin access:
+
+Raise a ticket through [Request Admin access](https://cpsprod1.service-now.com/esc?id=sc_cat_item&table=sc_cat_item&sys_id=47594fa61b861510729e42a7b04bcbb1).
+
+In the business justification field, specify that you require local admin access for development purposes. 
+
+Provide details about your project, etc.
+
+Please note that your admin account (e.g., username@CPSGOVUK.onmicrosoft.com) will grant you the permissions necessary to install software.
+
+## What development tools can I use?
+
+These are listed in the [starndards section](../../standards/archtecture/Accepted-Software/code-development.md)
+Required Software Tools
+
+A software audit is performed on the machines to help remind people when they forget to use the approved set of tools
+
+## What code repositories do I use, and how do I get access?
+
+As per the Technology COde of Practice we will "Publish ... code and use open source to improve transparency, flexibility and accountability". 
+The specific products are listed in the [starndards section](../../standards/archtecture/Accepted-Software/code-development.md). Currently this includes GitHub and GitLab.
+
+### Access to a GitHub repo: 
+
+Sign up for [a free GitHub account](https://github.com/signup?ref_cta=Sign+up&ref_loc=header+logged+out&ref_page=%2F&source=header-home) with your CPS email address.
```

Comment:
Small change here, to remove invalid tracking data from the url and account for people who already have a github account.

I would maybe change the text to:

```markdown
If you do not have a GitHub account, sign up for [a free account](https://github.com/signup) using your CPS email address.

If you already have a GitHub account, be sure to [add your CPS email address to it](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account).
```

Reply:
Good detail, have done